elements/bulbs-pollzone-map/bulbs-pollzone-map.js

```diff
@@ -0,0 +1,121 @@
+import { 
+  BulbsHTMLElement, 
+  registerElement 
+} from 'bulbs-elements/register';
+
+import _ from 'lodash';
+
+import * as d3 from 'd3';
+import topojson from 'topojson';
+import Datamap from 'datamaps';
+
+import './bulbs-pollzone-map.scss';
+
+class BulbsPollzoneMap extends BulbsHTMLElement {
+  attachedCallback () {
+    let $this = $(this).find('.map');
+    let $data = $this.data('map-data');
+
+    this.legendContainer = $(this).find('.legend')[0];
+    this.statesData = $data.votes;
+    this.questionsData = $data.questions;
+    this.setFills();
+    this.initMap();
+  }
+
+  setFills () {
+    this.fills = {
+      defaultFill: '#006B3A',
+    };
+
+    // add colors to keyed fills list
+    _.forEach([
```

Comment:
I know, I tried:

```
import chain from 'lodash/chain'
import isEmpty from 'lodash/isEmpty'
import forEach from 'lodash/forEach'
import sortBy from 'lodash/orderBy'
```

but the chained functions were having issues (toPairs, sortBy)